confignode/src/main/java/org/apache/iotdb/confignode/manager/ConfigManager.java

```diff
@@ -83,9 +86,38 @@ public int getDeviceGroupID(String device) {
     return hashExecutor.getDeviceGroupID(device);
   }
 
-  // TODO: Interfaces for metadata operations
+  /** Build ConfigNodeGroup ConsensusLayer */
+  private void setConsensusLayer(ConfigNodeConf config) {
+    // TODO: Support other consensus protocol
+    this.consensusImpl =
+        new StandAloneConsensus(
+            id -> {
```

Comment:
You can just use ` new StandAloneConsensus(id ->  new StandAloneStateMachine()`

Reply:
Fixed.